vSync/vs_server.lua

```diff
@@ -63,6 +63,11 @@ function isAllowedToChange(player)
 end
 
 RegisterCommand('freezetime', function(source, args)
+	TriggerClientEvent('vSync:freezeTime', source)
```

Comment:
Why did you add a unnecessary client event here? Why not just have a function that you call from both the registerCommand, as well as the new event which will check permissions and update time options accordingly? that saves an unnecessary call to the client and back to the server.